spyder/utils/programs.py

```diff
@@ -82,42 +82,28 @@ def is_program_installed(basename):
     """
     home = get_home_dir()
     req_paths = []
-    if sys.platform == 'darwin':
-        if basename.endswith('.app') and osp.exists(basename):
-            return basename
-
-        pyenv = [
-            osp.join('/usr', 'local', 'bin'),
-            osp.join(home, '.pyenv', 'bin')
-        ]
-
-        # Prioritize Anaconda before Miniconda; local before global.
-        a = [osp.join(home, 'opt'), '/opt']
-        b = ['anaconda', 'miniconda', 'anaconda3', 'miniconda3']
-        conda = [osp.join(*p, 'condabin') for p in itertools.product(a, b)]
+    if (sys.platform == 'darwin' and basename.endswith('.app')
+            and osp.exists(basename)):
+        return basename
 
-        req_paths.extend(pyenv + conda)
-
-    elif sys.platform.startswith('linux'):
+    if os.name == 'posix':
         pyenv = [
+            osp.join(home, '.pyenv', 'bin'),
             osp.join('/usr', 'local', 'bin'),
-            osp.join(home, '.pyenv', 'bin')
         ]
 
         a = [home, '/opt']
-        b = ['anaconda', 'miniconda', 'anaconda3', 'miniconda3']
-        conda = [osp.join(*p, 'condabin') for p in itertools.product(a, b)]
-
-        req_paths.extend(pyenv + conda)
-
-    elif os.name == 'nt':
+        b = ['mambaforge', 'miniforge',
+             'miniconda3', 'anaconda3', 'miniconda', 'anaconda']
```

Comment:
```suggestion
        b = ['mambaforge', 'miniforge3', 'miniforge',
             'miniconda3', 'anaconda3', 'miniconda', 'anaconda']
```

AFAIK, Python 3 Miniforge (but not Mambaforge) is installed to `miniforge3/` rather than `miniforge/` on *nix.

Reply:
Aah yes, you are correct.